includes/kp-functions.php

```diff
@@ -48,7 +48,7 @@ function get_klarna_customer( $customer_type ) {
 		$customer['klarna_access_token'] = $access_token;
 	}
 
-	return apply_filters( 'kp_set_customer_type', $customer );
+	return apply_filters( 'kp_set_customer_type', $customer_type );
```

Comment:
Sorry i wrote the previous comment a bit awkwardly, the 2nd parameter should be the `$customer` variable here, but add the `$customer_type` variable to the filter as 2nd parameter passed when using the filter, so in this case it should be the 3rd parameter here. So `apply_filters( 'kp_set_customer_type', $customer, $customer_type );`